js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -92,4 +95,35 @@ describe("OperatorSignInModal", () => {
     expect(view.getByText("Something went wrong")).toBeInTheDocument();
     expect(console.error).toHaveBeenCalledOnce();
   });
+
+  test("shows failure component when no operator is found after an NFC tap", () => {
+    const view = render(<OperatorSignInModal />);
+
+    jest.mocked(useNfc).mockReturnValueOnce({
+      result: { status: "success", data: "bad_serial" },
+      abortController: new AbortController(),
+    });
+    jest.mocked(findEmployeeByBadgeSerial).mockReturnValueOnce(undefined);
+
+    view.rerender(<OperatorSignInModal />);
```

Comment:
Could you destructure `rerender` from l100 so it matches the above cases?

Reply:
I forgot it was possible to do this from destructure syntax. Fixed in aabede7.